docs/en/latest/admin-api.md

```diff
@@ -1318,6 +1318,14 @@ Plugin resource request address: /apisix/admin/plugins/{plugin_name}
 
 The Plugin ({plugin_name}) of the data structure.
 
+### Request Arguments
+
+| Name      | Description                   | Default |
+| --------- | ----------------------------- | ------- |
+| subsystem | The subsystem of the Plugins. | http    |
+
+The plugin can be filtered on subsystem so that the ({plugin_name}) is searched in the subsystem passed through query params. Note that this query parameter is not available on `/apisix/admin/plugins/list`
```

Comment:
this query parameter is not available on `/apisix/admin/plugins/list`?
I think it's supported on `/apisix/admin/plugins/list`?

Reply:
It wasn't present present previously and was later added in the pull request. I removed the later comment.